main.go

```diff
@@ -42,6 +46,7 @@ import (
 var (
 	discoveryInterval = flag.Duration("discovery-interval", 30*time.Second, "How often to refresh discovery cache to pick up newly-installed resources")
 	informerRelist    = flag.Duration("cache-flush-interval", 30*time.Minute, "How often to flush local caches and relist objects from the API server")
+	caFile            = flag.String("ca", "", "Verify certificates of TLS-enabled secure servers using this CA bundle")
```

Comment:
Can we call the flag `ca-bundle-file` so it's clear what it does when looking at a command line in isolation?